lib/cli.rb

```diff
@@ -12,6 +13,17 @@ def sample
     puts presenter.present(recipe: recipe)
   end
 
+  def select_recipes(ingredients:)
+    matching_recipes = recipe_repository.recipes_with(ingredients: ingredients)
+
+    if matching_recipes.none?
+      puts "Could not find any recipes containing those ingredients"
+    else
+      selected_recipe = RecipeSelector.new(recipes: matching_recipes).select
+      puts presenter.present(recipe: selected_recipe)
```

Comment:
Possibly rename to `TerminalRecipeSelector`. Could also use as an instance variable to hide which one we use e.g.:
```
selected_recipe = recipe_selector.select_one_from(matching_recipes)
```